python/tskit/tables.py

```diff
@@ -30,6 +30,7 @@
 import warnings
 
 import numpy as np
+from jsonschema import validate
```

Comment:
I prefer importing the module and calling functions on it usually, as then it's clear in the code what functions are defined locally and which ones from from outside. So, when I see ``jsonschema.validate``, it's totally obvious, but if I read ``validate`` I'm not so sure.

Reply:
Yep, I agree - editor put this one in and I haven't done a full-on `diff` check.
https://github.com/tskit-dev/tskit/pull/491/commits/a0c2750c320ee78f239a2c13b2307f4d10712283